magda-web-client/src/helpers/previewData.js

```diff
@@ -23,6 +23,14 @@ export function getPreviewDataUrl(distributions: Array<ParsedDistribution>){
     const html = viewableDistribution.filter(d=> d.format.toLowerCase() === 'html');
     const rss = viewableDistribution.filter(d=> d.format.toLowerCase() === 'rss');
 
+    const geoFormat = ["csvGeoAu" , "wfs" , "wms" , "czml" , "kml"];
```

Comment:
Actually you can make it `csv-geo-` so that it'll work if csv-geo-au is generalized for other countries as well.